en/docs/setup/deployment-patterns.md

```diff
@@ -288,11 +288,123 @@ Following configurations need to be done to both WSO2 Identity Server nodes in o
-    ```
-    [server]
-    force_local_cache="true"
-    ```
+2. Configure caching.
+
+    !!! note
+        From WSO2 Identity Server 5.2.0 onwards, distributed caching is disabled and it is not recommended to use this due to many practical issues that are related to configuring and running distributed caching properly. WSO2 Identity Server employs **Hazelcast** as the primary method of implementing cluster messages while using distributed caching in a simple setup.
+
+    ??? info "About Caching"
+        <ul>
+            <li><b>Why caching</b></br>Caching is an additional layer on top of databases. It enables to keep the recently used data that are fetched from the database in local memory, so that for subsequent data requests instead of fetching from the database the data can be served from the local memory. Caching has certain advantages and disadvantages that you need to evaluate when deciding on your caching strategy.</li>
+            <li><b>Advantages</b>
+                <ul>
+                    <li>The load on the underlying database or LDAP is reduced as data is served from already fetched data in memory.</li>
+                    <li>Improved performance due to the reduced number of database calls for repetitive data fetching.</li>
+                </ul>
+            </li>
+            <li><b>Disadvantages</b>
+                <ul>
+                    <li>Coherency problems may occur when the data change is not immediately reflected on cached data if one node or an external system updates the database.</li>
+                    <li>Data in memory can become stale yet be served, e.g., serving data from memory while its corresponding record in the database is deleted.</li>
+                </ul>
+            </li>
+        </ul>
+
+    ??? tip "Caching in WSO2 Identity Server"
+        Historically WSO2 Identity Server used distributed caching to utilize the above-mentioned advantages as well as to minimize the coherence problem. However, in newer deployment patterns where the network is not tightly controlled, distributed caching fail in unexpected ways. Hence, we **no longer recommend using distributed caching**. Instead, it is **recommended to have local caches** (if required) and **cache invalidation messages** (if required) by considering the information given below.
+        <ul>
+            <li><b>The ForceLocalCache property</b></br>
+            When Hazelcast clustering is enabled certain caches act as distributed caches. The `force_local_cache` property in the `<IS_HOME>/repository/conf/deployment.toml` directory is there to mark that all the caches should act like local caches even in a clustered setup. (This is by default set to `true`).
+            ```
+            [server]
+            force_local_cache="true"
```

Comment:
It should be, (Set boolean value without quotes)
[server]
force_local_cache = true